portal/messaging/secure_chat.php

```diff
@@ -495,7 +495,7 @@ public function pingAction()
         return json_encode($onlines);
     }
 }
-
+$infoMsg = xlt('Click to activate and send to this recipient.');
```

Comment:
rec not escaping it here since you don't know if needs text() and attr(), and it's just less confusing to just do the escaping when it gets outputted to html